NASA-PDS/registry-mgr
# Added upgrade-dd command to Registry manager to upgrade data dictionary

## 🗒️ Summary
Added upgrade-dd command to Registry manager to upgrade data dictionary

**NOTE:** This branch includes changes from branch `versions` (https://github.com/NASA-PDS/registry-mgr/pull/46)

## ⚙️ Test Data and/or Report
* Run `registry-manager upgrade-dd -r`
* Check that `registry-dd` index was dropped and recreated and that data dictionary was loaded. You should see output similar to this:
```
[INFO] Deleting index registry-dd                                                                                                                      
[INFO] Creating index: registry-dd
[INFO] Schema: C:\tmp\registry-manager-4.5.0-SNAPSHOT\elastic\data-dic.json
[INFO] Shards: 1
[INFO] Replicas: 0
[INFO] Loading ES data file: C:\tmp\registry-manager-4.5.0-SNAPSHOT\elastic\data-dic-data.jar:dd.json
[INFO] Loaded 500 document(s)
[INFO] Loaded 1000 document(s)
[INFO] Loaded 1500 document(s)
[INFO] Loaded 2000 document(s)
[INFO] Loaded 2500 document(s)
[INFO] Loaded 3000 document(s)
[INFO] Loaded 3122 document(s)
```

## ♻️ Related Issues
https://github.com/NASA-PDS/registry/issues/43
